docs/api/events.rst

```diff
@@ -1377,3 +1377,85 @@ Voice Channel Status Update
 
     :param payload: The raw voice channel status update payload.
     :type payload: :class:`RawVoiceChannelStatusUpdateEvent`
+
+Voice Channel Effects
+---------------------
+.. function:: on_voice_channel_effect_send(event)
+
+    Called when a voice channel effect is sent.
+
+    .. versionadded:: 2.7
+
+    :param event: The voice channel effect event.
+    :type event: :class:`VoiceChannelEffectSendEvent`
+
+Soundboard Sound
+----------------
+.. function:: on_soundboard_sounds_update(before, after)
+
+    Called when multiple guild soundboard sounds are updated at once and they were all already in the cache.
+    This is called for example when a guild looses a boost level and some sounds become unavailable.
```

Comment:
loose (verb): set free; release.
lose (verb): be deprived of or cease to have or retain (something).
```suggestion
    This is called, for example, when a guild loses a boost level and some sounds become unavailable.
```